### ISSUE TITLE
Iowa Now experts list

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#5670]
# Story

As an Iowa Now website owner, I would like to be able to create a person list that has an exposed filter for areas of expertise so that media and other site visitors can quickly identify an expert by their area of expertise.

# Additional Information

Existing page - https://now.uiowa.edu/experts-list

Would like to be able to display the areas of expertise in the teaser view. They do not need the ability to link to each area of expertise. It sounded like we would prefer they use the website field to enter the url for the person's canonical profile page and then link the teaser to that page rather than viewing the full node on this site? If we go that route we would need to be able to display photo, first, last, email, phone, positions, areas of expertise and contact in the teaser. We may be able to avoid migrating the person's about info since it doesn't seem to be used on the staff listings https://now.uiowa.edu/staff-list and I doubt we want to output that in a teaser. They would lose the ability to link to a related article on these experts profiles.

Do we need a person type to differentiate between an expert and Iowa Now staff or all experts faculty? Could we just make this determination based on if the person has areas of expertise listed?

# Proposed Solution
- Custom view display to expose filter for research areas and output research areas in teaser display.
- Move "Research Areas" to display in the teaser at /admin/structure/types/manage/person/display/teaser.

**Comment by briand44** [comment]
Can we use our existing Research areas field for this? If so, can we overwrite the label to be "Areas of expertise"?

**Comment by briand44** [comment]
Does this relate to https://github.com/uiowa/uiowa/issues/3804

**Comment by joewhitsitt** [comment]
> Can we use our existing Research areas field for this? If so, can we overwrite the label to be "Areas of expertise"?

We've modified the field label before https://github.com/uiowa/uiowa/pull/4142, but I wonder if they are open to leaving it as-is and adjusting their terminology for this.

**Comment by joewhitsitt** [comment]
>Does this relate to https://github.com/uiowa/uiowa/issues/3804

Yes if we want to address this with default functionality instead of a custom view page. Another workaround could be using the default SiteNow People views page and marking `hide on lists` for anyone that shouldn't be on that list initially.

**Comment by briand44** [comment]
List of experts will link to profile page on Iowa Now.

**Comment by briand44** [comment]
Grooming conversation - get feedback on if experts are all faculty or how to distinguish between staff and experts. Also feedback on research areas vs areas of expertise terminology. Custom view display to expose filter for research areas and output research areas in teaser display.

**Comment by briand44** [comment]
They said they are ok with using "Research areas" terminology.

Not all experts are faculty so it sounds like we will want to use an "Experts" tag and assign that tag when migrating people that are currently of the type "Faculty experts".

**Comment by bspeare** [comment]
It seems like https://github.com/uiowa/uiowa/issues/5670#issuecomment-1240960140 makes the most sense here.  Except that the "Hide on lists" will not allow them to create a custom OSC Staff page using the People List block.   Is there a better way to do this that doesn't recreate the SiteNow People views page?

**Comment by bspeare** [comment]
@joewhitsitt we discussed this a little more this morning.  What do you think about adding the ability to filter `/admin/config/sitenow/sitenow-people` by tag as a way of moving forward with https://github.com/uiowa/uiowa/issues/5670#issuecomment-1266036429?

<img width="1690" alt="Screen Shot 2022-10-04 at 10 20 27 AM" src="https://user-images.githubusercontent.com/1036433/193859916-329b111a-7d1d-4ac1-9bc7-c6423b35f27c.png">

**Comment by briand44** [comment]
If the plan is to create the OSC staff page by using the People list block and filtering by staff, how do we handle the scenario where there is a non-osc staff member that is an expert. They would have a tag for expert so they would appear in the experts page but they would also appear on the OSC staff listing page, correct? In those cases do they just not designate that person with a person type? Or do we create an OSC staff tag as well on migration and use that to determine the OSC staff? Trying to think through all scenarios here. I do like avoiding building a custom experts view if we can use the People page.

**Comment by joewhitsitt** [comment]
Brian keeps editing his comment so it is hard to follow but...

I do think we should tag/type folks that we migrate over.

However, the more I think about it and from what I understand, I don't think SiteNow People view page is best suited for the Experts list. SiteNow People also controls the path at which all people on the website reside. So if they want to maintain the path at `/experts-list` then all people will be nested under that.

"What's one more custom view block here?" Experts list block that has the research filter on it showing only people with type/tag expert. When exposed filters is possible on People list block we can switch it and clean up.

**Comment by bspeare** [comment]
Thanks @joewhitsitt.  That sounds like a good plan.  I just wanted to see if there was an opportunity to expand the functionality of SiteNow People before we do this work.

**Comment by briand44** [comment]
>Brian keeps editing his comment so it is hard to follow but...

Sorry, my comment is really unrelated to whether we use SiteNow people for experts but more on how we are displaying staff list.

>However, the more I think about it and from what I understand, I don't think SiteNow People view page is best suited for the Experts list. SiteNow People also controls the path at which all people on the website reside. So if they want to maintain the path at /experts-list then all people will be nested under that.

That is a good point I didn't think about.

**Comment by joewhitsitt** [comment]
Filtering by tag on SiteNow People view page whether it is user exposed or not seems cool, but the original purpose of the view was to be a full a-z utility view and not a subset of content. Tags aren't currently displayed on people so if user exposed, I don't think tags are clear to the user what they plan on getting.
